gen/sam/sam_sa8.py

```diff
@@ -15,13 +15,19 @@ class SA8Generator(AirDefenseGroupGenerator):
     price = 55
 
     def generate(self):
-        self.add_unit(
-            AirDefence.Osa_9A33_ln,
-            "OSA",
-            self.position.x,
-            self.position.y,
-            self.heading,
+        num_launchers = 2
```

Comment:
http://www.ausairpower.net/APA-9K33-Osa.html#mozTocId178145 suggests this should be 4

Reply:
I did not want to represent accurate launcher sites, since DCS does not represent realistic flight packages either (which is the reason a SA-10 does not have 15 Launchers). My intention was to make it consistent for campaign designers. The feedback for this approach in discord was good.